### PR TITLE
Update activities departments relation to a many to many

### PR DESCRIPTION
**Pull request by feliciofilipe**

### CODE REVIEW

**Comment by MarioRodrigues10** [comment]
It is not possible to update or remove departments and speakers from activities.

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
It is not possible to update or remove departments and speakers from activities.

**Comment by feliciofilipe** [comment]
@MarioRodrigues10 review it again plz :pray:

File: lib/atomic/activities/activity.ex [hunk 1]
```diff
@@ -5,10 +5,12 @@ defmodule Atomic.Activities.Activity do
...
 
   alias Atomic.Activities
+  alias Atomic.Activities.ActivityDepartment
   alias Atomic.Activities.ActivitySpeaker
   alias Atomic.Activities.Enrollment
   alias Atomic.Activities.Session
   alias Atomic.Activities.Speaker
+  alias Atomic.Departments
   alias Atomic.Departments.Department
   @required_fields ~w(title description
```
Comment (MarioRodrigues10) [inline lib/atomic/activities/activity.ex:15]: Please remove department_id from @required_fields

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
In order to fix this error when creating a new activity ```unknown field `:department_id` given to cast.```.

File: lib/atomic/activities/activity.ex [hunk 2]
```diff
@@ -23,11 +25,9 @@ defmodule Atomic.Activities.Activity do
     field :minimum_entries, :integer
     field :enrolled, :integer, virtual: true
 
-    many_to_many :speakers, Speaker,
-      join_through: ActivitySpeaker,
-      on_replace: :delete
+    many_to_many :speakers, Speaker, join_through: ActivitySpeaker, on_replace: :delete
 
-    belongs_to :department, Department
```
Comment (MarioRodrigues10) [inline lib/atomic/activities/activity.ex:30]: Due to this change ``` field `department_id` in `where` does not exist in schema Atomic.Activities.Activity ```, so it isn't possible to edit or create a department

Comment (MarioRodrigues10) [reply lib/atomic/activities/activity.ex:30]: This wasn't resolved

Comment (MarioRodrigues10) [reply lib/atomic/activities/activity.ex:30]: This wasn't resolved

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Review by MarioRodrigues10 — APPROVED** [review]
Approved because the department form issue is solved in #266

**Pull request merged**